actors/account/src/lib.rs

```diff
@@ -32,8 +32,6 @@ fil_actors_runtime::wasm_trampoline!(Actor);
 pub enum Method {
     Constructor = METHOD_CONSTRUCTOR,
     PubkeyAddress = 2,
-    // Deprecated in v10
-    // AuthenticateMessage = 3,
```

Comment:
I don't think we should remove these (in any actor). They are useful documentation of method numbers that have been used, and relevant to interpreting old blockchain history. These lines aren't costing us anything to retain.